scripts/links/generate.js

```diff
@@ -0,0 +1,49 @@
+const fse = require('fs-extra');
+const chalk = require('chalk');
+const fm = require('front-matter');
+const get = require('lodash/get');
+const { resolve } = require('path');
+
+const DOCS_FOLDER = '../docs';
+const OUTPUT = './links/links.js';
+const FRONT_MATTER_KEY = 'platform_key';
+const FRONT_MATTER_ID = 'id';
+const URL_ROOT = 'DOCS_URL_ROOT';
+
+const traverse = async (path) => {
+  const startingPaths = await fse.readdir(path, { withFileTypes: true });
+  const links = await Promise.all(
+    startingPaths.map(async (pathEntry) => {
+      const resPath = resolve(path, pathEntry.name);
+      return pathEntry.isDirectory() ? traverse(resPath) : generateLink(resPath);
+    }),
+  );
+  return Array.prototype
+    .concat(...links)
+    .filter(Boolean)
+    .join('');
```

Comment:
wanna template in a comment at the top with a comment mentioning this file? :)

Reply:
@hlminh2000 i made it medium scary